packages/ui-components/src/components/Panel/Panel.stories.tsx

```diff
@@ -4,11 +4,12 @@
  */
 
 import React from "react"
-import { Panel } from "./index.js"
-import { PanelBody } from "../PanelBody/index.js"
-import { ContentAreaWrapper } from "../ContentAreaWrapper/index.js"
-import { ContentArea } from "../ContentArea/index.js"
-import { PortalProvider } from "../../deprecated_js/PortalProvider/PortalProvider.component.js"
+
```

Comment:
We shoudl fix this types all story :) make it typed

Reply:
Still a draft, working on it locally.